rfcs/0029-ckb-hardfork/0029-ckb-hardfork.md

```diff
@@ -0,0 +1,87 @@
+---
+Number: "0029"
+Category: Standards Track
+Status: Proposal
+Author: Wanbiao Ye
+Organization: Nervos Foundation
+Created: 2021-05-25
+---
+
+# CKB Hardfork
+
+## Abstract
+
+This document describes the first hard fork of ckb.
```

Comment:
This document only describes the syscalls upgrade, not the first fork of CKB.

Reply:
It's better to mention the related RFCs and add the backlink to the related RFCs so when developers read the old RFC, they known there are updates to the syscalls.

Reply:
> This document only describes the syscalls upgrade, not the first fork of CKB.

This is the @xxuejie 's suggestion. He hopes that there will be an RFC describing all the modifications of hardfork, syscalls is only a part of it.

Reply:
But for now, you can still describe it as VM & syscall changes for the next hardfork. Later we can expand this to cover all CKB changes(and change the wording here), or use a separate RFC